tests/test_transforms/test_timestamp/test_fourier_transform.py

```diff
@@ -1,10 +1,12 @@
 from copy import deepcopy
 
 import numpy as np
+import numpy.testing
```

Comment:
Is it used here?

Reply:
I use `np.testing.assert_allclose(transform_values, expected_values, atol=1e-12)`.

Reply:
you already have imported numpy